README.md

```diff
@@ -22,41 +18,26 @@ Follow the steps below to get this application fully functional and running usin
 
 1. Create a [Stytch](https://stytch.com/) account. Once your account is set up a Project called "My first project" will be automatically created for you.
 
-2. Within your new Project, navigate to [SDK configuration](https://stytch.com/dashboard/sdk-configuration), and make the following changes:
-
-   - Under **Test environment**, Click **Enable SDK in test**.
-   - Under **Authorized applications** add the domain `http://localhost:3000` if it isn't already there.
-   - Within the **Email Magic Links** drawer, toggle on **Enable the LoginOrCreate Flow**.
-     
-     <img width="400" alt="SDK Email Magic Links" src="https://user-images.githubusercontent.com/100632220/217053215-8c369de8-7828-4ad6-ac88-a50918520fc3.png">
-
-   - Toggle on **OAuth**.
-     
-     <img width="400" alt="SDK OAuth" src="https://user-images.githubusercontent.com/100632220/217053483-e757d1aa-af18-4af3-a476-45860ca3065f.png">
+2. Within your new Project, navigate to [SDK configuration](https://stytch.com/dashboard/sdk-configuration), and click **Enable SDK**.
 
-3. Navigate to [Redirect URLs](https://stytch.com/dashboard/redirect-urls), and add `http://localhost:3000/authenticate` as the types **Login** and **Sign-up**.
-   
-   <img width="400" alt="Redirect URLs" src="https://user-images.githubusercontent.com/100632220/217983021-d8bf6fff-6a68-4e94-bffd-d062e69c8817.png">
+3. Navigate to [OAuth](https://stytch.com/dashboard/oauth), and set up login for Google in the Test environment.
```

Comment:
Do we need this/should we re-word since Google OAuth is configured by default now in Test?

Reply:
Yeah, fair to slightly adjust now that its config'd by default.